shex/_Patient.shex

```diff
@@ -0,0 +1,45 @@
+PREFIX cmumps: <http://hokukahu.com/schema/cmumpss#>
```

Comment:
Funny filename, starting with an underscore: shex/_Patient.shex
Is this intentional?